libs/components/indicators/src/lib/modules/alert/alert.component.scss

```diff
@@ -58,6 +59,12 @@
     height: 32px;
   }
 
+  &:not(.sky-alert-danger) {
+    ::ng-deep .sky-icon-inverse {
```

Comment:
Should we make this a first class feature of the icon stack component? The ability to set a color for the top icon? We then could use a css custom property to manage that in the icon stack component.